docs/usage/managing-versions.md

```diff
@@ -0,0 +1,149 @@
+---
+description: >-
+  Learn more about the Tuist built-in version management and how you can
+  leverage it to ensure reproducible environments.
+---
+
+# Managing versions
+
+When adding a dependency between your project and third-party tools, it’s crucial that the dependency is reproducible. If the dependency is not the same across all the environments, it might produce different results with the same input. Have you ever heard the well-known _“it works for me”_? That’s often the result of having non-reproducible environments.
+
+If a tool is written in Ruby or Javascript, we can use dependency managers like [Bundler](https://bundler.io) and [NPM](https://www.npmjs.com), which support not only pining the version of the dependencies but executing the right version that your project is pinned to. For instance, if you run `bundle exec cocoapods`, Bundler makes sure that the version of CocoaPods that you are executing is the version that is specified in your `Gemfile`.
+
+Although the [Swift Package Manager](https://swift.org/package-manager/) supports defining dependencies to be linked from your packages, it doesn’t support defining tools your project depends on. In other words, there isn’t a `swift exec tuist`. The closest option is [Mint](https://github.com/yonaskolb/mint), a non-official tool that aims to solve that problem. However, it means a dependency with another tool that should be installed in the environment, probably with [Homebrew](https://brew.sh), and whose versioning is out of our control. Besides introducing some inconvenience, it might result in a non-reproducible environment.
+
+**Tuist opted for a simple, reproducible and transparent version management approach that is built into Tuist.** If you followed the [Getting Started](https://tuist.io/guides/1-getting-started) guide when you installed Tuist, you installed a tool called `tuistenv` that got renamed into `tuist`. Try executing the following command:
+
+```bash
+tuist --help-env
+```
+
+It’ll execute help on `tuistenv` which prints the list of commands that are available to manage the Tuist environment. In the following sections, we’ll go through each of these commands, explain what they do, and what they are for.
+
+## Commands
+
+### tuist local
+
+When executed with no arguments, `tuist local`, it prints the list of versions that are available locally:
+
+```bash
+$ tuist local
+
+The following versions are available in the local environment:
+- 0.4.0
+```
+
+If you execute the same command passing a version as an argument, it pins the directory where you are running the command from to the given version:
+
+```text
+$ tuist local 0.4.0 
```

Comment:
Commands in the getting started didn't use `$` as a prefix for each bash line, do you think it's worth adding for consistency?

Reply:
We don't really need them. I'll remove them.